@here/harp-mapview/lib/DecodedTileHelpers.ts

```diff
@@ -422,65 +491,239 @@ export function getMaterialConstructor(technique: Technique): MaterialConstructo
  * @param technique technique from where params are copied
  * @param material target material
  * @param level optional, tile zoom level for zoom-level dependent props
- * @param skipExtraProps optional, skipped props
+ * @param skipExtraProps optional, skipped props.
  */
-export function applyTechniqueToMaterial(
+function applyTechniqueToMaterial(
     technique: Technique,
     material: THREE.Material,
     level: number,
     skipExtraProps?: string[]
 ) {
-    Object.getOwnPropertyNames(technique).forEach(propertyName => {
+    // Remove transparent color from the firstly processed properties set.
+    const baseColorPropName = getBaseColorPropName(technique);
+    const hasBaseColor = baseColorPropName && baseColorPropName in technique;
+    const genericProps = Object.getOwnPropertyNames(technique).filter(propertyName => {
         if (
             propertyName.startsWith("_") ||
             BASE_TECHNIQUE_NON_MATERIAL_PROPS.indexOf(propertyName) !== -1 ||
             DEFAULT_SKIP_PROPERTIES.indexOf(propertyName) !== -1 ||
             (skipExtraProps !== undefined && skipExtraProps.indexOf(propertyName) !== -1)
         ) {
-            return;
+            return false;
+        }
+        // Omit base color and related transparency attributes if its defined in technique.
+        if (
+            baseColorPropName === propertyName ||
+            (hasBaseColor && TRANSPARENCY_PROPERTY_KEYS.indexOf(propertyName) !== -1)
+        ) {
+            return false;
         }
         const prop = propertyName as keyof typeof technique;
         const m = material as any;
         if (typeof m[prop] === "undefined") {
-            return;
+            return false;
         }
-        let value = technique[prop];
-        if (isInterpolatedProperty(value) || Expr.isExpr(value)) {
-            value = getPropertyValue(value, level);
+        return true;
+    });
+
+    // Apply all other properties (even colors), but not transparent (base) ones.
+    genericProps.forEach(propertyName => {
+        const value = technique[propertyName as keyof Technique];
+        if (value !== undefined) {
+            applyTechniquePropertyToMaterial(material, propertyName, value, level);
         }
-        applyTechniquePropertyToMaterial(material, prop, value);
     });
+
+    // Finally apply base (possibly transparent) color itself, modifying material.opacity and
+    // material.transparent attributes too.
```

Comment:
This comment is not accurate anymore. The transparent attribute stays untouched

Reply:
Fixed.